File: Cemetery/Models/Burial.cs
/* Модель для изменения */

using Newtonsoft.Json;
using System;

namespace CemeteryApp.Models
{
    public class Burial
    {
        public int Id { get; set; }
        public string Surname { get; set; }
        public string Name { get; set; }
        public string Patronymic { get; set; }
        public DateTime DeathDate { get; set; }
        public DateTime BirthDate { get; set; }
        public DateTime BurialDate { get; set; }
        public string[] imgs { get; set; }
        // [JsonIgnore]
        public DateTime ChangeDate { get; set; }
        // [JsonIgnore]
        public DateTime CreateDate { get; set; }
        public string FuneralUrn { get; set; }

        //
        [JsonIgnore]
        public int PlaceId { get; set; }
        //[JsonIgnore]
        public Place Place { get; set; }
        //
        //[JsonIgnore]
        public Relative Relative { get; set; }
        //
        //[JsonIgnore]
        public Other Other { get; set; }
    }
}

File: Cemetery/ClientApp/src/app/burial.ts
import { Place } from "./place";

export class Burial {
    // burial info
    id?: number;
    surname?: string;
    name?: string;
    patronymic?: string;
    deathDate?: Date;
    birthDate?: Date;
    burialDate?: Date;
    imgs?: string[];
    changeDate?: Date;
    createDate?: Date;
    funeralurn?: string;
    placeId?: number;

    // other info
    certificateNumber?: string;
    registrationOffice?: string;
    dateOfReference?: Date;
    registrationAddress?: string;
    archivedNumber?: number;
    cremationPlace?: string;
    graveDepth?: number;
    ditcherSurname?: string; // ???????
    soilType?: string;

    // relative info
    relativeFullName?: string;
    address?: string;
    contacts?: string;

    //cemetery?: string;
    //sector?: number;
    //place?: number;
    //row?: number;

    //area?: number;
    // status?: string;
    // burialType: string;



    //datacr?: string;
    // place: Place;
}